src/app/banners/Banner.tsx

```diff
@@ -1,11 +1,10 @@
 import React, { useEffect } from 'react';
 import { X } from 'phosphor-react';
-import Hearts from 'assets/images/banner/hearts.png';
-import NeonBlur from 'assets/images/banner/neonBlur.png';
+import NeonBlur from 'assets/images/banner/infinity.svg';
 import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
 
 const Banner = () => {
-  const [showLifetimeBanner, setShowLifetimeBanner] = React.useState(false);
+  const [showLifetimeBanner, setShowLifetimeBanner] = React.useState(true);
```

Comment:
I would add the opening and closing logic and the possible variables outside the Banner component, so you can reuse it for each banner you need for the different campaigns by creating a wrapper.